test/Constraints/common_type.swift

```diff
@@ -28,9 +28,33 @@ func f(_: Double) -> Y { return Y() }
 
 func testCallCommonType() {
   // CHECK: overload set choice binding $T{{[0-9]+}} := (Int) -> X
-  // CHECK-NEXT: (common result type for $T{{[0-9]+}} is Int)
+  // CHECK-NEXT: (considering -> (_const $T2) -> $T3 applicable fn $T1
```

Comment:
Let’s not use concrete numbers for type variables here because they might change, you could use `[[<name>:$T[0-9]+]]` to define a name for a particular variable and use that name to reference it elsewhere in test. Take a look at how SILGen/IRGen tests do that

Reply:
Same in other tests, also indents are fine I think.